alpha/lib/enums/PermissionName.php

```diff
@@ -275,4 +275,6 @@ interface PermissionName extends BaseEnum
 
 	const FEATURE_ANALYTICS_PERSISTENT_SESSION_ID = 'FEATURE_ANALYTICS_PERSISTENT_SESSION_ID';
 	const FEATURE_LIVE_ANALYTICS_DASHBOARD = 'FEATURE_LIVE_ANALYTICS_DASHBOARD';
+
+	const FEATURE_LIVE_ENTRIES_IN_FEED = 'FEATURE_LIVE_ENTRIES_IN_FEED';
```

Comment:
@TalDubovKaltura  I would change the name of the feature to "FEATURE_INCLUDE_LIVE_ENTRIES_IN_FEED"

Reply:
Done.